packages/dbadmin/src/controllers/DBAdminApi.ts

```diff
@@ -55,7 +55,12 @@ export class DBAdminApiImpl implements DBAdminApi {
     createDatabase = async(request: e.Request, response: e.Response) => {
         logger.requestLog(` * createDatabase request received, with body of ${request.headers["content-length"]} bytes`)
         const result = await this.ctx.dbAdminApiWorker.createDatabase(CREATE_DATABASE_SQL)
```

Comment:
I would consider renaming this one to `createDbResult`

Reply:
Removed the variable, as the call either is ok, or throws an exception. Result is never useful.